Failing test: Rules Management - Rule Bulk Actions Integration Tests - ESS Env - Trial License.x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action·ts - Rules Management - Rule Bulk Action API @ess @serverless @skipInServerless perform_bulk_action @skipInServerlessMKI fill gaps run action should trigger the backfilling of the gaps for the rules in the request

A test failed on a tracked branch

```
JestAssertionError: expect(received).toEqual(expected) // deep equality

- Expected  - 1
+ Received  + 6

- Array []
+ Array [
+   Object {
+     "gte": "2025-06-19T18:28:37.968Z",
+     "lte": "2025-06-19T18:29:37.967Z",
+   },
+ ]
    at forEach (perform_bulk_action.ts:2719:52)
    at Array.forEach (<anonymous>)
    at Context.<anonymous> (perform_bulk_action.ts:2717:25)
    at processTicksAndRejections (node:internal/process/task_queues:105:5)
    at Object.apply (wrap_function.js:74:16) {
  matcherResult: {
    actual: [ [Object] ],
    expected: [],
    message: '\x1B[2mexpect(\x1B[22m\x1B[31mreceived\x1B[39m\x1B[2m).\x1B[22mtoEqual\x1B[2m(\x1B[22m\x1B[32mexpected\x1B[39m\x1B[2m) // deep equality\x1B[22m\n' +
      '\n' +
      '\x1B[32m- Expected  - 1\x1B[39m\n' +
      '\x1B[31m+ Received  + 6\x1B[39m\n' +
      '\n' +
      '\x1B[32m- Array []\x1B[39m\n' +
      '\x1B[31m+ Array [\x1B[39m\n' +
      '\x1B[31m+   Object {\x1B[39m\n' +
      '\x1B[31m+     "gte": "2025-06-19T18:28:37.968Z",\x1B[39m\n' +
      '\x1B[31m+     "lte": "2025-06-19T18:29:37.967Z",\x1B[39m\n' +
      '\x1B[31m+   },\x1B[39m\n' +
      '\x1B[31m+ ]\x1B[39m',
    name: 'toEqual',
    pass: false
  }
}
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/70557#0197895d-1730-45c4-9a75-53441c8b41b4)

<!-- kibanaCiData = {"failed-test":{"test.class":"Rules Management - Rule Bulk Actions Integration Tests - ESS Env - Trial License.x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action·ts","test.name":"Rules Management - Rule Bulk Action API @ess @serverless @skipInServerless perform_bulk_action @skipInServerlessMKI fill gaps run action should trigger the backfilling of the gaps for the rules in the request","test.failCount":15}} -->

--- a/x-pack/solutions/security/test/security_solution_api_integration/config/services/detections_response/rules/get_gaps_by_rule_id.ts
+++ b/x-pack/solutions/security/test/security_solution_api_integration/config/services/detections_response/rules/get_gaps_by_rule_id.ts
@@ -18,6 +18,7 @@ export const getGapsByRuleId = async (
   const response = (await supertest
     .post(routeWithNamespace(`/internal/alerting/rules/gaps/_find`, namespace))
     .set('kbn-xsrf', 'foo')
+    .set('x-elastic-internal-origin', 'kibana')
     .send({
       rule_id: ruleId,
       start,

--- a/x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts
+++ b/x-pack/solutions/security/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts
@@ -92,8 +92,7 @@ export default ({ getService }: FtrProviderContext): void => {
   const createWebHookConnector = () => createConnector(getWebHookAction());
   const createSlackConnector = () => createConnector(getSlackAction());
 
-  // Failing: See https://github.com/elastic/kibana/issues/224615
-  describe.skip('@ess @serverless @skipInServerless perform_bulk_action', () => {
+  describe('@ess @serverless perform_bulk_action', () => {
     beforeEach(async () => {
       await deleteAllRules(supertest, log);
       await esArchiver.load('x-pack/platform/test/fixtures/es_archives/auditbeat/hosts');

